pkg/cluster/ansible/import.go

```diff
@@ -113,3 +137,237 @@ func SSHKeyPath() string {
+	for _,v := range hosts {
+		if len(allVarSame) == 0 || v.Vars[key] != allVarSame[0]  {
+			allVarSame = append(allVarSame,v.Vars[key])
+		}
+	}
+	return allVarSame
+}
+
+// parse config files
+func ParseConfigFile( cfgfile string ) (map[string]interface{} , error) {
+	srvConfigs :=make(map[string]interface{})
+	sectionRegex := regexp.MustCompile(`^\[([^:\]\s]+)(?::(\w+))?\]\s*(?:\#.*)?$`)
+	reader, err := os.Open(cfgfile)
+	if err != nil {
+		return nil, err
+	}
+	defer reader.Close()
+	bufreader := bufio.NewReader(reader)
+	scanner := bufio.NewScanner(bufreader)
+	prefixkey := ""
+	for scanner.Scan() {
+		line := strings.TrimSpace(scanner.Text())
+		if strings.HasPrefix(line, ";") || strings.HasPrefix(line, "#") || line == "" {
+			continue
+		}
+		matches := sectionRegex.FindAllStringSubmatch(line, -1)
+		if matches != nil {
+			prefixkey = matches[0][1]
+			continue
+		} else if strings.HasPrefix(line, "[") && strings.HasSuffix(line, "]") {
+			return nil,fmt.Errorf("Invalid section entry: '%s'. Please make sure that there are no spaces in the section entry, and that there are no other invalid characters", line)
+		} else {
+			keyval := strings.SplitN(strings.ReplaceAll(line," ",""),"=",2)
+
+			if len(keyval) == 1 {
+				return nil,fmt.Errorf("Bad key=value pair supplied: %s", line)
+			}
+
+			if strings.Trim(keyval[1], "\"") == "" ||strings.Contains(keyval[0],"host") || strings.Contains(keyval[0],"port") {
+				continue
+			}
+
+			if prefixkey != "" {
+				keyval[0] = prefixkey+"."+keyval[0]
+			}
+
+			if keyval[0] == "server.labels" {
+				val := strings.Split(strings.ReplaceAll(strings.ReplaceAll(strings.Trim(strings.Trim(keyval[1],"{"),"}"),"\"","")," ",""),",")
+				label_val := make(map[string]interface{})
+				for i:=0;i<len(val);i++ {
+					kv := strings.SplitN(val[i],"=",2)
+					label_val[kv[0]] = kv[1]
+				}
+				srvConfigs[keyval[0]] = label_val
+			} else if keyval[0] == "replication.location-labels" {
+               val := strings.Split(strings.ReplaceAll(strings.ReplaceAll(strings.Trim(strings.Trim(keyval[1],"["),"]"),"\"","")," ",""),",")
+               srvConfigs[keyval[0]] = val
+			} else {
+				srvConfigs[keyval[0]] = strings.ReplaceAll(keyval[1],"\"","")
+			}
+		}
+	}
+	return srvConfigs, nil
+}
+
+
+// Load config files to clusterMeta ,just include tidbservers, tikvservers and pdservers
+func LoadConfig(clsName string ,cls *spec.ClusterMeta) error {
+	prefixkey := ""
+	var err error
+	Configs :=  make(map[string]interface{})
+	gConfigs := make(map[string]interface{})
+	initflag := false
+	for i := 0 ; i < len(cls.Topology.TiDBServers); i++ {
+		prefixkey = "tidb"
+		srv := cls.Topology.TiDBServers[i]
+		fname := spec.ClusterPath(clsName,spec.AnsibleImportedConfigPath,fmt.Sprintf("%s-%s-%d.toml",prefixkey,srv.Host, srv.Port))
+
+		Configs , err  = ParseConfigFile(fname)
+		if initflag == false {
+			gConfigs = Configs
+			initflag = true
+		}
+		// make Configs containts global Config variables
```

Comment:
We have `Merge2Toml()` in `pkg/cluster/spec/server_config.go` that could be helpful for this purpose.